tmdb-api/tmdb_api_solution.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+"""tmdb-api_solution
+
+Automatically generated by Colab.
+
+Original file is located at
+    https://colab.research.google.com/drive/1zAKT_SAwNoqC9Zn6F7i6qE4-HfemCTaJ
+
+# **#1 and #2**
+"""
+
+!pip install tmdbsimple
+
+import tmdbsimple as tmdb
+import requests
+import json
+import pandas as pd
+tmdb.API_KEY = 'b3ca90af8d6988fd6220149ce6a3235f'
+
+search = tmdb.Search()
+starWars = search.movie(query = 'Star Wars')
+starWars
+
+starWars.keys() # I just wanted to get a better view of the dictionary, so I
+# investigated the keys first
+# The keys are 'page', 'results', 'total_pages', and 'total_results'
+
+"""# **#3**"""
+
+starWars_df = pd.DataFrame(data = starWars['results']) # Data Frame
```

Comment:
You'll need to get the data from all pages. This is just the first page of data (hint: see the keys you got from starWars.keys() and look into the "page" parameter for search.movie()).